src/App.jsx

```diff
@@ -1,8 +1,73 @@
 import './App.css'
-
+import Button from "./components/buttons.jsx";
+import Product from "./components/product.jsx";
+import bag1 from "././assets/bag_1.png"
+import bag2 from "././assets/bag_2.png"
+import bag3 from "././assets/bag_3.png"
+import bag4 from "././assets/bag_4.png"
+import story from "././assets/our_story.png"
+import brand from "././assets/brand.png"
+import Tile from "./components/tile.jsx";
 function App() {
-  return (
+  return (<>
+
       <h1>Handbags & Purses</h1>
+      <nav>
+          <Button
+              text="to the collection"
+              disabled={false}/>
+          <Button
+              text="shop all bags"
+              disabled={false}/>
+          <Button
+              text="pre-orders"
+              disabled={true}/>
+      </nav>
+          <main>
+              <Product
+                  label={"Best seller"}
```

Comment:
Bij de properties die je doorgeeft in het Product-element heb je gebruik gemaakt van {} en “”, het is de bedoeling om ze niet allebei te gebruiken. Bijvoorbeeld {"Best seller"} moet zijn "Best seller" of {Best seller}.